.gitignore

```diff
@@ -0,0 +1 @@
+venv/
```

Comment:
Could we add a new line at the end of the file?

Reply:
Done, thank you